staging/traefik/README.md

```diff
@@ -247,6 +273,8 @@ The following table lists the configurable parameters of the Traefik chart and t
 | `forwardAuth.authResponseHeaders`      | Set authentication response headers                                                                                                  |  `[]`                                     |
 | `initCertJobImage`                     | Set the Docker image to be used for updating the cert-manager certificate with the load balancer SANs                        | `mesosphere/kubeaddons-addon-initializer:v0.0.7`  |
 | `extraConfigEntrypoints`               | Extra configurations for the entrypoints.                                                                                    | None                                              |
+| `extraVolumeMounts`      | Any extra volumes mounts to define for the Traefik container                                                                                                                                                                                                               | `[]`                                                                                                                      |
```

Comment:
not yours, but could you add `extraSSLEntrypoints` here which is something we introduced